### PR TITLE
rfc: instance/group normalization

### PR DESCRIPTION
**Pull request by igorsafo**
# Description

[A Link to the rendered document](https://github.com/igorsafo/oneDNN/blob/rfcs/rfcs/20230315-instance-normalization/README.md)

### CODE REVIEW

**Comment by igorsafo** [comment]
@jgong5 @yiqianglee Could you please provide feedback from PyTorch side?

**Comment by Zantares** [comment]
Glad to hear that oneDNN will support InstanceNorm primitive. Note that Intel Extension for TensorFlow (ITEX) also has the core op and it's implemented by Loop + BN.

**Comment by yimeisun123** [comment]
Thanks for planning to add support for this use case.
"tiny" comment, I think the model is called 3D U-Net.

**Comment by igorsafo** [comment]
> Thanks for planning to add support for this use case. "tiny" comment, I think the model is called 3D U-Net.

Thanks, updated the naming to 3D U-Net

File: rfcs/20230315-instance-normalization/README.md [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+| [6, 128, 6, 7, 7]      | 0.000012 | 0.000133    | 0.000053              | 11.08333333              |
+
+
+The performance is collected on a 28 core Intel(R) Xeon(R) Platinum 8280 CPU using the following command:
+```sh
+$ numactl --physcpubind=0-27 -l ./build/examples/primitives-instance-normalization-cpp
+```
+
+A loop over Batch normalization is slower than a single call to either layer or
+batch normalization. One of import differences is Instance normalization has
```
Comment (dzarukin) [inline rfcs/20230315-instance-normalization/README.md:285]:
```suggestion
batch normalization. One of important differences is Instance normalization has
```

**Review by dzarukin — APPROVED** [review]

File: rfcs/20230315-instance-normalization/README.md (hunk 1 above)
Comment (igorsafo) [reply rfcs/20230315-instance-normalization/README.md]: fixed, thanks
